features/yield-calculator/YieldCalculator.tsx

```diff
@@ -139,12 +139,21 @@ const YieldCalculator = () => {
                 />
               </FormInput>
             </Grid>
-            <Grid item xs={4}>
-              <Box pt={4}>
-                <Typography variant="h6">
-                  Yearly APR: {prettyPercentage(yieldAmount)}%
-                </Typography>
-              </Box>
+            <Grid item md={4} xs={12}>
+              <Hidden only={["sm", "xs"]}>
+                <Box pt={4}>
+                  <Typography variant="h6">
+                    Yearly APR: {prettyPercentage(yieldAmount)}%
+                  </Typography>
+                </Box>
+              </Hidden>
+              <Hidden only={["md", "lg", "xl"]}>
```

Comment:
What's the point of `Hidden`?

Reply:
depending on the screen size there needs to be different padding. `<Hidden only={["md", "lg", "xl"]}>` will hide the code block on `md` `lg` and `xl` screen sizes.